packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -606,6 +606,9 @@ export class TreeCheckout implements ITreeCheckoutFork {
 				}
 			},
 			clone: (forkedBranch: TreeBranch) => {
+				/**
+				 * To clone `RevertibleAlpha` in batch, use {@link cloneRevertiblesInBatch}.
```

Comment:
Does this need to be a multi-line comment?

Reply:
I could make it single line. I also want to make sure that if this is the best place to include the comment. 

While providing the utility function separately, I wish the function to be discoverable.

Reply:
This comment would probably be better as an `@remark` on the `RevertibleAlpha` type itself under the `clone` attribute.